package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.5.1",
   "description": "Extensions that augments several Mozilla websites (Bugzilla, DXR, Searchfox, MozReview) with code coverage information.",
   "dependencies": {
+    "eslint-plugin-mozilla": "^0.9.0",
     "web-ext": "^2.4.0"
   },
```

Comment:
`eslint` should be defined as a peerDependency: https://eslint.org/docs/developer-guide/working-with-plugins#peer-dependency. `eslint-plugin-mozilla` should probably be in devDependencies.
We should also move `web-ext` to `devDependencies`, but in a separate PR.